src/components/button-group/button-group.scss

```diff
@@ -131,3 +139,11 @@
         @include shared_input-select-picker.looks-disabled;
     }
 }
+
+limel-badge {
+    margin-right: 0.25rem;
```

Comment:
Really good catch! An alternative would be using `pointer-events: none;` perhaps more correct, because the badge is just ornamental/visual and shouldn't react to clicks. By using `cursor: pointer;` we will get the same result, but if I was reading the styles code, I would assume that the badge is an interactive element; but it's not a button or link, so the browser doesn't add the pointer style automatically, that's why the write wrote it manually. So it's a bit misleading that way.

Reply:
Thank you both, I'm gonna add `pointer-events: none;`